test/serde/serde_helpers.py

```diff
@@ -1813,8 +1813,8 @@ def make_gradfn(**kwargs):
     alice, bob = kwargs["workers"]["alice"], kwargs["workers"]["bob"]
     t = torch.tensor([1, 2, 3])
 
-    x_share = t.share(alice, bob, requires_grad=True)
-    y_share = t.share(alice, bob, requires_grad=True)
+    x_share = t.share(alice, bob, crypto_provider=alice, requires_grad=True)
```

Comment:
That's bad practice to put the same worker in crypto_provider than in *workers 
because this means that this guy can decode everything, which we don't want.  crypto_provider should be a distinct party, this actually might be worth an extra test!